infra/charts/turing/values.yaml

```diff
@@ -96,9 +96,22 @@ turing:
 
   clusterConfig:
     # -- (bool) Configuration to tell Turing API how it should authenticate with deployment k8s cluster
-    # By default, Turing API expects to use a remote k8s cluster for deployment and to do so,
-    # it requires cluster credentials to be stored in Vault's KV Secrets store.
+    # By default, Turing API expects to use a remote k8s cluster for deployment and to do so, it requires
+    # cluster access configurations to be configured as part of values.yaml
     useInClusterConfig: false
+    environmentConfigPath: "environments.yaml"
+    ensemblingServiceK8sConfig: 
```

Comment:
nit: Can we have a comment that links to `environments-dev.yaml` file which shows the expected values for this config?

Similar comment for `environmentConfigs` below.